gnome/gnome-radio/Portfile

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                gnome-radio
+version             0.1.2
+set branch          [join [lrange [split $version .] 0 1] .]
+
+categories          gnome
+platforms           darwin
+license             GPL-3+
+maintainers         {gnome.org:ole @oleaamot} openmaintainer
+description         GNOME Radio
+long_description    GNOME Radio is the Public Network Radio Software for Accessing Free Audio Broadcasts from the Internet on GNOME.
+homepage            http://www.gnomeradio.org/
+master_sites        http://www.gnomeradio.org/${branch}/
+distname            ${name}-${version}
```

Comment:
This is the default value for `distname` so this line should be removed.

Reply:
OK, fixed.